src/classes/CAO_Constants_API.cls

```diff
@@ -0,0 +1,104 @@
+/*
+    Copyright (c) 2018, Salesforce.org
+    All rights reserved.
+    
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+    
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/**
+* @author Salesforce.org
+* @date 2018
+* @group API
+* @group-content ../../ApexDocContent/API.htm
+* @description API class that exposes CAO (Contacts and Organizations) Constants and helper methods.
+*/
+global class CAO_Constants_API {
+
+    /*******************************************************************************************************
+    * @description One-to-One value for system processor field
+    */ 
+    global static final String ONE_TO_ONE_PROCESSOR {
+        get {return CAO_Constants.ONE_TO_ONE_PROCESSOR;}
+        private set;
+    }
+
+    /*******************************************************************************************************
+    * @description One-to-One org type value
+    */ 
+    global static final String ONE_TO_ONE_ORGANIZATION_TYPE {
```

Comment:
Just noticing the `final` keyword. If these are `private set;`, is there a reason to use `final`? I'm asking because as global methods, we won't be able to ever change these once they're included in a package.

Reply:
Fair point, I'll remove final.